File: multiple_inheritence.py
class A:
    name = None
    def __init__(self,name) -> None:
        self.name = name
    def show(self):
        print(self.name)
            
class B():
    age = 10
class C(A,B):
    def __init__(self, name) -> None:
        A.__init__(name)
    def show(self):
        print(self.age)
obj = B("Rocky")
obj.show()

File: multiple_inheritence.cpp
#include <bits/stdc++.h>
using namespace std;
class A{
    public:
    string name;
    void show(){
        cout<<name<<"from class a"<<endl;
    }
};
class B{
    public:
    string age;
};
class C:public A,public B{
    public:
    C(int c){
        this->age =c;
    }
    void show_age(){
        cout<<age<<endl;
    }
};
int main()
{
    ios_base::sync_with_stdio(0);
    cin.tie(0);
    cout.tie(0);

    C obj(10);
    obj.show_age();

    return 0;
}